src/com/yetanalytics/lrs_admin_ui/language.cljs

```diff
@@ -0,0 +1,185 @@
+(ns com.yetanalytics.lrs-admin-ui.language)
+
+(def language
+  {;;Footer   
+   :footer.attribution     {:en-US "Yet Analytics Inc."}
+   :footer.license         {:en-US "Licensed under the Apache 2.0 License"}
+   :footer.contribute      {:en-US "Contribute on "}
+   :footer.contact-note    {:en-US "Contact us to learn about Enterprise Support options."}
+   :footer.nav.logout      {:en-US "LOGOUT"}
+   :footer.nav.browser     {:en-US "BROWSER"}
+   :footer.nav.accounts    {:en-US "ACCOUNTS"}
+   :footer.nav.credentials {:en-US "CREDENTIALS"}
+   ;;Header
+   :header.welcome         {:en-US "Welcome, "}
+   :header.logout          {:en-US "Logout"}
+   :header.nav.credentials {:en-US "Credentials Management"}
+   :header.nav.accounts    {:en-US "Account Management"}
+   :header.nav.browser     {:en-US "Statement Browser"}
+   :header.nav.monitor     {:en-US "LRS Monitor"}
+   :header.nav.data        {:en-US "Data Management"}
+   :header.nav.reactions   {:en-US "Reactions"}
+   ;;Login Page
+   :login.title        {:en-US "LOGIN"}
+   :login.username     {:en-US "Username"}
+   :login.password     {:en-US "Password"}
+   :login.login-button {:en-US "LOGIN"}
+   :login.trouble      {:en-US "Trouble logging in? See provided documentation about account management or contact your system administrator.123"}
+   :login.oidc-button  {:en-US "OIDC LOGIN"}
+   ;;Credentials Page 
+   :credentials.title       {:en-US "CREDENTIALS MANAGEMENT"}
+   :credentials.tenant.add  {:en-US "ADD NEW CREDENTIALS"}
+   :credentials.tenant.number  {:en-US "Number of Credentials: "}
+   :credentials.tenant.key  {:en-US "Api Key"}
+   :credentials.key.aria {:en-US "Show/Hide Api Key Details"}
+   :credentials.key.permissions {:en-US "Permissions "}
+   :credentials.key.secret {:en-US "API Key Secret"}
+   :credentials.key.hide {:en-US "Hide"}
+   :credentials.key.show {:en-US "Show Secret Key"}
+   :credentials.key.permissions.save {:en-US "Save"}
+   :credentials.key.permissions.cancel {:en-US "Cancel"}
+   :credentials.key.edit {:en-US "Edit"}
+   :credentials.key.delete {:en-US "Delete"}
+   :credentials.key.delete.confirm {:en-US "Are you sure?"}
+   ;;Accounts Page
+   :accounts.delete {:en-US "Delete"}
+   :accounts.delete.confirm {:en-US "Are you sure?"}
+   :accounts.password.update {:en-US "Update Password"}
+   :accounts.new.username {:en-US "Username:"}
+   :accounts.new.username.note {:en-US "Username must be %d or more alphanumeric characters"}
+   :accounts.new.password {:en-US "Password:"}
+   :accounts.new.password.note {:en-US "Password must be %d or more characters and contain uppercase, lowercase, numbers, and special characters (%s). Be sure to note or copy the new password as it will not be accessible after creation."}
```

Comment:
One thing to note is that format strings might not be enough for multi-lingual support, since different languages have different word orders, so you'll end up with the ordered `%d` positions in the wrong places. Not a huge deal at the moment, but something to keep in mind for the future.

Reply:
ahhh super interesting point, but kinda unavoidable from this angle.